### ISSUE TITLE
"Git branch:" blank [MASTER]

### ISSUE THREAD

**Issue by axemann** [mescon/Muximux#141]
Hi guys,

I have an issue where in the Settings screen, the "Git branch:" drop-down box is blank and has no choices available:

![image](https://cloud.githubusercontent.com/assets/17078534/23327354/10d4a6f2-fad0-11e6-9533-9e1d927850ff.png)

This occurs whether it's a fresh clone of the repo or not.

I'm not sure if this is expected behavior, but based on the screenshots in README.md I'm betting not. :-)

Anyone else encounter this?  I'm running it with NginX in a FreeNAS (FreeBSD) jail.

Thanks!

**Comment by axemann** [comment]
Okay, I think I ended up fixing it on my own.  I did a parallel fresh install into an Ubuntu 16.04 VM and everything appeared to display and populate properly (branch in UI and sha entries in settings.ini.php).  After comparing the installed packages between Ubuntu and FreeNAS (FreeBSD) I found that FreeBSD requires 'php70-openssl' in order to properly parse the Github URL.  This may be a fix for #137 as well.
